docs/source/core_services/historians/index.rst

```diff
@@ -42,6 +42,15 @@ All historians support the following settings:
         # Defaults to 30
         "max_time_publishing": 30.0,
 
+        # Limit how far back the historian will keep data in days.
+        # Partial days supported via floating point numbers.
+        # A historian must implement this feature for it to be enforced.
+        "history_limit_days": 366,
```

Comment:
Would history_limit_days override storage_limit_gb or would the storage limit get precedence. Or does it depend on implementation ?

Reply:
Technically it depends on the implementation. The intent is to enforce both of them if you set both of them. As one restriction will be a super set of the other (they both must delete the oldest data first) any implementation should just enforce one and then the other in any order and the net effect will be the same. 

I chose to do the time limit first as it would most likely exceed the size limit in database size reduction and it faster to enforce. In most cases if they are both set the GB size check would pass over that loop without removing any data.